src/mlpack/tests/imputation_test.cpp

```diff
@@ -0,0 +1,323 @@
+/**
+ * @file imputation_test.cpp
+ * @author Keon Kim
+ *
+ * Tests for data::Imputer class
+ */
+#include <sstream>
+
+#include <mlpack/core.hpp>
+#include <mlpack/core/data/dataset_mapper.hpp>
+#include <mlpack/core/data/map_policies/increment_policy.hpp>
+#include <mlpack/core/data/map_policies/missing_policy.hpp>
+#include <mlpack/core/data/imputer.hpp>
+#include <mlpack/core/data/imputation_methods/custom_imputation.hpp>
+#include <mlpack/core/data/imputation_methods/listwise_deletion.hpp>
+#include <mlpack/core/data/imputation_methods/mean_imputation.hpp>
+#include <mlpack/core/data/imputation_methods/median_imputation.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include "test_tools.hpp"
+
+using namespace mlpack;
+using namespace mlpack::data;
+using namespace std;
+
+BOOST_AUTO_TEST_SUITE(ImputationTest);
+
+/**
+ * Make sure a CSV is loaded correctly.
```

Comment:
I think this comment may not describe what are the test case doing.
Either remove it, or make it more expressive.